src/TTextEdit.cpp

```diff
@@ -290,6 +293,7 @@ void TTextEdit::updateScreenView()
         updateScrollBar(mpBuffer->mCursorY);
```

Comment:
As I am currently seeing the code it is (from new line 291):
```cpp
    mScreenHeight = visibleRegion().boundingRect().height() / mFontHeight;
    if (!mIsLowerPane) {
        updateScrollBar(mpBuffer->mCursorY);
    }
    int currentScreenWidth = visibleRegion().boundingRect().width() / mFontWidth;

    } else {
        mScreenWidth = currentScreenWidth;
    }
        // Note the values in the "parent" Host instance - for the UPPER pane
        // so that they are available for NAWS:
        if (!mIsLowerPane) {
            mpHost->setScreenDimensions(mScreenWidth, mScreenHeight);
        }
    } else {
        mScreenWidth = currentScreenWidth;
    }

    if (mpHost->mAutoWrap) {
        mpBuffer->setWrapAt(mScreenWidth);
    }

    mOldScrollPos = mpBuffer->getLastLineNumber();
}
```

Either my web browser is really messed up - it could be, GitHub insists on using bleeding edge code and that is not fully standards compliant and breaks on browsers that are not trying to be Chrome-a-likes (i.e. not Chrome/FireFire/Edge?) - or there is something messed up with that chunk...

Reply:
*\*Sigh\** - another case where I wasn't seeing the whole picture. Sorry but there is nothing to see here - just move along please... 🙄